bruh_mqtt_multisensor_github/bruh_mqtt_multisensor_github.ino

```diff
@@ -35,11 +37,11 @@
 
 
 /************ WIFI and MQTT INFORMATION (CHANGE THESE FOR YOUR SETUP) ******************/

```

Comment:
Thanks much for the addition. One request though, can we leave lines 38-42 as they were. I like having the explanation of what to type in rather than saying redacted. Otherwise, looks great. Cheers!